tools/wavetool.py

```diff
@@ -205,6 +210,17 @@ def analyze_wav_wov(wave, fs):
         sys.exit(1002)
     print('wave analysis result: PASSED')
 
+def analyze_wav_thdn(wave, fs_wav):
+    wave = normalize(wave)
+    check_rms(wave)
+    thdn = calc_thdn(wave, fs_wav, cmd.freq[0])
+    result = thdn < cmd.threshold
+    print('THD+N of specified wave is %s dB' % thdn)
+    if not np.all(result):
+        print('THD+N too high, wave analysis result: FAILED')
+        sys.exit(1003)
```

Comment:
why the exit code is 1003

Reply:
The exit code is selected, start from 1000, 1001 for smart_amp failure, 1002 for wov failure and 1003 for thdn failure. this should not matter, I think.

Reply:
Define some constants and return them instead of hardcoding 1003?

Reply:
Use binary bits coding for multiple fails?

Reply:
@singalsu The exit code just denote failure, I just want to use different code to denote different failure. this script is called from bash, we only care it is PASS of FAIL. Btw, I have printed some very useful logs.